Agile_Essay.tex

```diff
@@ -0,0 +1,97 @@
+% Please do not change the document class
+\documentclass{scrartcl}
+
+% Please do not change these packages
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+
+\title{How can introverts best adapt to the agile philosophy?}
+
+\subtitle{COMP150 - Agile Development Practice}
+
+\author{1608351}
+
+\begin{document}
+
+\maketitle
+
+\abstract{A variety of personalities and skill-sets in individuals within a team tends to enhance performance, therefore it is important not to disregard those who struggle with agile principles. Finding methods that give individuals the autonomy to align their working style with agile, may help ensure diversity within teams.  This paper explores how individuals, with personality types that have been found to have a negative relationship with agile preference, can best adapt to the agile philosophy.}
+
+\section{Introduction}
+
+Agile processes are widely used in the Game Development industry, they encourage a focus on self-organising teams along with regular reflection, and communication \cite{Agile}. Due to the scope involved in game development tasks, working within teams, rather than individually, is necessary. However, such focus on teamwork and communication can be problematic for introverts. A correlation has been found between an individual's personality type, based on the Five-Factor model (FFM), and their preference for agile processes \textit{`The data supports a positive relationship between extraversion and agile preference as well as openness and agile preference'} \cite{BishopDeokar}. FFM marks personality traits across a continuum of five distinct strands, namely: Extraversion, Openness, Agreeableness, Neuroticism, and Conscientiousness. Although individuals are rarely able to be placed in one category exclusively, there is usually a tendency towards certain factors in FFM, and therefore, this tool can still be useful in managing effects of personality on the workplace.

```

Comment:
*Always* include page numbers for a quotation. You can do this with the cite command using square brackets. For example: `\cite[p. 143]{BishopDeokar}`.